src/application.ts

```diff
@@ -89,14 +90,17 @@ export default class Application {
   async buildOperationResponse(
     data: Resource | Resource[] | void
   ): Promise<OperationResponse> {
-    const included = flatten(await this.extractIncludedResources(data)).filter(
-      Boolean
-    );
+    const included = flatten(await this.extractIncludedResources(data))
+    const uniqueIncluded = Array.from(new Set(included
```

Comment:
@spersico This can be written in spread notation:

```ts
const uniqueIncluded = [...new Set(included)].map(...).map(...)
```

Reply:
done